beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -59,216 +59,218 @@ function LikeButton() {
   const [liked, setLiked] = React.useState(false);
 
   if (liked) {
-    return 'You liked this!';
+    return 'आपको यह पसंद आया!';
   }
 
   return React.createElement(
     'button',
     {
       onClick: () => setLiked(true),
     },
-    'Like'
+    'पसंद करें'
   );
 }
 ```
 
-### Step 4: Add your React Component to the page {/*step-4-add-your-react-component-to-the-page*/}
+### स्टेप 4: पेज पर अपना React कौम्पोनॅन्ट ऐड करें {/*step-4-add-your-react-component-to-the-page*/}
 
-Lastly, add two lines to the bottom of **like_button.js**. These two lines of code find the `<div>` you added to your HTML in the first step and then display the "Like" button React component inside of it.
+अंत में, **like_button.js** के नीचे दो पंक्तियाँ ऐड करें। कोड की ये दो पंक्तियाँ पहले स्टेप में आपके द्वारा अपने HTML में ऐडे गए `<div>` को ढूंढती हैं और फिर उसके अंदर "पसंद करें" बटन React कौम्पोनॅन्ट प्रदर्शित करती हैं।
```

Comment:
`ऐडे` => `ऐड किये गए`

Reply:
@Anishpras here `गए` is added two times at line 77. Can you update that. seems a typo from my end which added the suggestion